server/api/db/sqldb/db.py

```diff
@@ -1797,7 +1810,7 @@ def _get_function_uid(
                 )
             return tag_function_uid
 
-    def _delete_functions(self, session: Session, project: str):
+    def _delete_project_functions(self, session: Session, project: str):
         for function_name in self._list_project_function_names(session, project):
             self.delete_function(session, project, function_name)
```

Comment:
```suggestion
        function_names = self._list_project_function_names(session, project)
        self.delete_functions(session, project, function_names)
```
check if function_names inside delete functions

Reply:
checked in side `_delete_multi_objects`